.github/workflows/rust-audit.yml

```
@@ -0,0 +1,22 @@
+name: Rust audit
+
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install cargo-audit
+      run: cargo install --force cargo-audit
+    - name: Generate lockfile
+      run: cargo generate-lockfile
+    - name: Perform audit
+      run: cargo audit
```

It might be nicer to use the [actions-rs/audit-check](https://github.com/marketplace/actions/rust-audit-check) action rather than running these shell commands.

```suggestion
      - uses: actions-rs/audit-check@v1
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
```

---

I bundled the suggestions into a squashed commit
I opted for using the pinned version `actions-rs/audit-check@v1.2.0`, since there was a version tag `v1", as I was suspecting the `actions-rs/audit-check@v1` would not use the `1.2.0` version. Can you confirm or correct my suspicion @RadicalZephyr ?